site/_docs/configuration.md

```diff
@@ -571,11 +571,11 @@ file or on the command-line.
 # Where things are
 source:       .
 destination:  ./_site
-plugins_dir:  ./_plugins
-layouts_dir:  ./_layouts
-data_dir:     ./_data
-includes_dir: ./_includes
-collections:  null
+plugins_dir:  _plugins
+layouts_dir:  _layouts
+data_dir:     _data
+includes_dir: _includes
```

Comment:
The definition for `destination` seems correct. It's not relative to `source` the way the others are.
```
    DEFAULTS = Configuration[{
      # Where things are
      "source"            => Dir.pwd,
      "destination"       => File.join(Dir.pwd, "_site"),
      "plugins_dir"       => "_plugins",
      "layouts_dir"       => "_layouts",
      "data_dir"          => "_data",
      "includes_dir"      => "_includes",
      "collections"       => {},
```